<launch>

  <!-- map nodes -->
  <node pkg="map_server" name="map_server" type="map_server" args="$(find rapp_map)/map/523.yaml" output="screen"/>
  <node pkg="rapp_map" name="rapp_setup_tf" type="rapp_setup_tf.py" output="screen"/>

  <!-- Core agent vision nodes -->
  <node pkg="rapp_robot_agent" name="acore_camera_server" type="acore_camera_server.py" output="screen"/>
  <node pkg="rapp_robot_agent" name="acore_transform_server" type="acore_transform_server.py" output="screen"/>
  <node pkg="rapp_robot_agent" name="acore_adyn_qrcode_server" type="adyn_qrcode_server.py" output="screen"/>    <!-- QR code detection server-->

  <!-- Core agent move node -->
  <node pkg="rapp_robot_agent" name="acore_move_server" type="acore_move_server.py" output="screen"/>
 
  <!-- Core agent navigation node -->
  <node pkg="rapp_robot_agent" name="acore_estimator_server" type="acore_estimator_server.py" output="screen"/>
 
  <!-- Kalman filter node -->
  <include  file="$(find robot_localization)/launch/nao_ekf.launch" />


</launch>
